ang/test/civicase/case/directives/case-overview.directive.spec.js

```diff
@@ -240,17 +245,29 @@
       });
     });
 
+    describe('when using pagination', () => {
+      beforeEach(() => {
+        element.isolateScope().setPageTo(5);
+      });
+
+      it('displays the content for the clicked page', () => {
+        expect(element.isolateScope().pageObj.num).toBe(5);
+        expect(CaseManagementWorkflow.getWorkflowsList).toHaveBeenCalled();
+      });
+    });
+
     /**
      * Initialise directive.
      *
      * @param {string} params the case type category name.
      */
     function compileDirective (params) {
+      $scope.currentCaseCategory = params.caseTypeCategory;
       $scope.caseFilter = {
-        'case_type_id.case_type_category': params.caseTypeCategory,
-        case_type_id: params.caseTypeID
+        'case_type_id.case_type_category': params.caseTypeCategory
       };
-      element = $compile('<civicase-case-overview case-filter="caseFilter"></civicase-case-overview>')($scope);
+
+      element = $compile('<civicase-case-overview case-filter="caseFilter" current-case-category="currentCaseCategory"></civicase-case-overview>')($scope);
```

Comment:
please remove `current-case-category="currentCaseCategory"`.

Also, when these strings start getting long we can use ` to split them into multiple lines.